src/Provider/Doctrine/Auditing/Transaction/TransactionHydrator.php

```diff
@@ -80,30 +80,35 @@ private function hydrateWithScheduledCollectionUpdates(Transaction $transaction,
 
         /** @var PersistentCollection $collection */
         foreach (array_reverse($uow->getScheduledCollectionUpdates()) as $collection) {
-            /** @var object $owner */
```

Comment:
Now that phpstan can pick up the correct types, it's yelling at me that `$owner` can be null (so can `$mapping`), which I guess it can, in some weird situations, where Doctrine code decides to return null. I think it's better to explicitly handle that situation  (line 85 below, `if ($owner && $this->provider->isAudited($owner)) {`), rather than forcing an incorrect type using inline comments?

Reply:
You're right and I would have done the same. Though, I prefer explicit conditions regarding `null` 
=> `if (null !== $owner) {` instead of `if ($owner) {`

Reply:
Yeah I saw you like strict types. Given the `PersistentCollection::getOwner()` returns `object|null`, the `$owner` can't be any other falsy value than `null`. I think `null !== $owner` is more verbose for no real reason. In saying so, it's your say, and it will work either way. Updated.